lib/sidekiq/instrument/middleware/server.rb

```diff
@@ -6,11 +6,15 @@ class ServerMiddleware
 
     def call(worker, job, queue, &block)
       StatsD.increment(metric_name(worker, 'dequeue'))
+      DogStatsD.increment('sidekiq.dequeue', worker_dog_options(worker))
 
-      StatsD.measure(metric_name(worker,'runtime'), &block)
+      StatsD.measure(metric_name(worker, 'runtime'), &block)
+      DogStatsD.time('sidekiq.runtime', worker_dog_options(worker)) { block }
```

Comment:
This will run the `&block` code twice!!
Run the block code only once, measure the timing, and emit it to StatsD and DogStatsD

Reply:
Hey @mokpro, I'm still pretty green with Ruby: would I measure the timing with `Time.now` calls surrounding the block running (and then recording the difference in time as milliseconds), or would it be better to require benchmark and do `Benchmark.realtime { block }`?  Or is there another better way?

Thanks for reviewing and catching this!